storage_service/locations/api/resources.py

```diff
@@ -772,6 +772,7 @@ class Meta:
             "package_type",
             "size",
             "status",
+            "stored",
```

Comment:
Can we use a different name here? "stored" is a reserved keyword since mysql 5.7.6 ( https://dev.mysql.com/doc/refman/5.7/en/keywords.html )

Reply:
Maybe `stored_at`?